lib/src/license_detection/crc32_test.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:test/test.dart';
+
+import 'crc32.dart';
+
+void main() {
+  final file = File('./lib/src/license_detection/crc32_random.json');
+  var z = jsonDecode(file.readAsStringSync()) as Map<String, dynamic>;
+
+  /// The random vectors were generate using this python code and then
+  /// verified with [GO crc32 package][]
+  /// ```python
+  /// import zlib
+  /// import json
+  /// import string
+  /// import random
+  /// f = open('crc.json', 'w')
+  /// jsonJ = {}
+  /// for i in range(1000):
+  ///     z = i % 20
+  ///     if z < 1:
+  ///         z = 1
+  ///     res = ''.join(random.choices(string.printable, k=z))
+  ///     crc = zlib.crc32(bytes(res, 'utf8'))
+  ///     jsonJ[res] = crc
+  /// json.dump(jsonJ, f)
+  /// ```
+  /// [GO crc32 package] : https://golang.org/pkg/hash/crc32/
+  group('Random vectors', () {
```

Comment:
```suggestion
  group('crc32 - random vectors', () {
```

Reply:
Just so that when you see the line in a failed test somewhere it says `crc-32` somewhere :D